manifests/prometheus/scripts/generate-manifest.sh

```diff
@@ -11,10 +11,22 @@ for i in "${PAAS_CF_DIR}"/manifests/prometheus/operations.d/*.yml; do
   opsfile_args+="-o $i "
 done
 
+alerts_opsfile_args=""
+for i in "${PAAS_CF_DIR}"/manifests/prometheus/alerts.d/*.yml; do
+  alerts_opsfile_args+="-o $i "
+done
+
+vars_files=""
```

Comment:
I think it would be more consistent to call this `varsfile_args`, which might be different enough to `VARS_FILES` that you don't need to disable the shellcheck warning anymore